methods/incidentReports.coffee

```diff
@@ -21,15 +21,22 @@ Meteor.methods
     return newId
 
   # similar to editIncidentReport, but allows you to set a single field without changing any other existing fields.
-  updateIncidentReport: (incident) ->
+  updateIncidentReport: (incident, fieldsToRemove=[]) ->
     _id = incident._id
     delete incident._id
     user = Meteor.user()
     if not Roles.userIsInRole(user._id, ['admin'])
       throw new Meteor.Error("auth", "User does not have permission to edit incidents")
     incident.modifiedByUserId = user._id
     incident.modifiedByUserName = user.profile.name
-    res = Incidents.update({_id: _id}, {$set: incident})
+    updateOperators =
+      $set: incident
+    if fieldsToRemove.length
+      fieldsToUnset = {}
+      fieldsToRemove.forEach (field) ->
```

Comment:
I think either method would be fine. However, editIncidentReport ought to prevent users from submitting properties that are maintained by the application like added_____. Those properties should not be passed from the client. The editIncidentReport method should look up the prior incident report and add the application properties to the submission in order to preserve them.